### ISSUE TITLE
TF 2.0 AttributeError: module 'tensorflow._api.v2.autograph.experimental' has no attribute 'do_not_convert'

### ISSUE THREAD

**Issue by HuzuniV2** [tensorflow/datasets#802]
**Short description**
Its not possible to import tensorflow_datasets using Tensorflow 2.0.
This error appears every time: `AttributeError: module 'tensorflow._api.v2.autograph.experimental' has no attribute 'do_not_convert'`

**Environment information**
* Operating System: `win10` and `Ubunto 18`
* Python version: `3.6.8`
* `tensorflow-datasets`/`tfds-nightly` version: `tensorflow-datasets-1.1.0`
* `tensorflow`/`tensorflow-gpu`/`tf-nightly`/`tf-nightly-gpu` version: `Tensorflow 2.0.0-alpha0`

import tensorflow_datasets as tfds

```
>>> import tensorflow_datasets as tfds
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\Utilizador\Miniconda3\envs\venv\lib\site-packages\tensorflow_datasets\__init__.py", line 46, in <module>
    from tensorflow_datasets.core import tf_compat
  File "C:\Users\Utilizador\Miniconda3\envs\venv\lib\site-packages\tensorflow_datasets\core\__init__.py", line 28, in <module>
    from tensorflow_datasets.core.dataset_info import DatasetInfo
  File "C:\Users\Utilizador\Miniconda3\envs\venv\lib\site-packages\tensorflow_datasets\core\dataset_info.py", line 53, in <module>
    from tensorflow_datasets.core.features import top_level_feature
  File "C:\Users\Utilizador\Miniconda3\envs\venv\lib\site-packages\tensorflow_datasets\core\features\__init__.py", line 27, in <module>
    from tensorflow_datasets.core.features.features_dict import FeaturesDict
  File "C:\Users\Utilizador\Miniconda3\envs\venv\lib\site-packages\tensorflow_datasets\core\features\features_dict.py", line 28, in <module>
    from tensorflow_datasets.core.features import top_level_feature
  File "C:\Users\Utilizador\Miniconda3\envs\venv\lib\site-packages\tensorflow_datasets\core\features\top_level_feature.py", line 27, in <module>
    class TopLevelFeature(feature_lib.FeatureConnector):
  File "C:\Users\Utilizador\Miniconda3\envs\venv\lib\site-packages\tensorflow_datasets\core\features\top_level_feature.py", line 45, in TopLevelFeature
    @tf.autograph.experimental.do_not_convert()
AttributeError: module 'tensorflow._api.v2.autograph.experimental' has no attribute 'do_not_convert'
```

**Comment by us** [comment]
Thanks for reporting! Can you check with `Tensorflow 2.0.0-beta1` version.

**Comment by HuzuniV2** [comment]
> Thanks for reporting! Can you check with `Tensorflow 2.0.0-beta1` version.

That does solve the issue yes. Much appreciated.

**Comment by RahulR432** [comment]
Is it possible to import tensorflow_datasets using any other TF 2.x version other than the beta.

**Comment by Conchylicultor** [comment]
@RahulR432 `tf.compat.v2.autograph.experimental` should be available in all version of TF 2.0.0, including the last stable 2.1. https://www.tensorflow.org/api_docs/python/tf/autograph/experimental/do_not_convert
